src/meshcat/commands.py

```diff
@@ -44,6 +44,17 @@ def lower(self):
             u"matrix": list(self.matrix.T.flatten())
         }
 
+class CaptureImage:
+    __slots__ = ["save_path"]
+    def __init__(self, save_path=""):
+        self.save_path = save_path
```

Comment:
It seems like `save_path` isn't actually used anywhere, since the image is returned as a PIL.Image (a good idea) rather than being written to disk. Can we just remove `save_path` entirely?

Reply:
Yes, let's remove it.

Reply:
Commit c7a4c73 adresses this, though I had to insert a UTF-8 encoded empty string to make sure the length of the `frames` arg is right